src/tutorial/output.md

```diff
@@ -1,15 +1,140 @@
 # Output
 
+## Printing "Hello World"
+
+```rust
+println!("Hello World");
+```
+
+Well, that was easy.
+Great, onto the next topic.
+
+## Using println
+
+You can pretty much print all the things you like
+with the `println!` macro.
+This macro has some pretty amazing capabilities,
+but also a special syntax.
+It expects you to write,
+as a first parameter,
+a literal string that with placeholders,
```

Comment:
This sentence is a bit clunky.

"It expects you to write a string literal as the first parameter, that contains placeholders that will be filled in [...]"